apps/web/lib/trpc/routers/workspace/create.ts

```diff
@@ -23,6 +23,8 @@ export const createWorkspace = t.procedure
       });
     }
 
+    const subscriptions = defaultProSubscriptions();
```

Comment:
Non related to the issue bur helpful for contributors, as they will probably have not added a stripe setup to the repo when they try to create a workspace. Stripe config isn't mentioned in the Get Started and it's also not shown as optional in the env.example (Would be cool to create an issue so that this is improve this in case it's needed)

So, whenever they try to create a new workspace, the trpc call fails as 'stripeEnv' (inside defaultProSubscription) throws an error. The contributor ends up having an ORG created in clerk that is not linked to a db workspace.